conan.cmake

```diff
@@ -318,15 +320,15 @@ function(conan_cmake_install)
         else()
             set(CONAN_BUILD_POLICY ${CONAN_BUILD_POLICY} --build=${ARG})
         endif()
-    endforeach()
+    endforeach()   
```

Comment:
Please clean up the PR by trimming trailing whitespace (here and down below).

Reply:
yeah, my fault, I didn't want to commit this, removed